iOS Client/ArticlesController/MBArticleDetailViewController.swift

```diff
@@ -61,6 +63,34 @@ class MBArticleDetailViewController: UIViewController, WKNavigationDelegate {
         MBStore.sharedStore.dispatch(PopCurrentNavigation())
     }
     
+    func configureShareButton() {
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.action, target: self, action: #selector(self.shareArticle(sender:)))
+    }
+    
+    @objc func shareArticle(sender: AnyObject) {
```

Comment:
nice so this is all we do? Just give the activity view controller a link and it's good to go?

Reply:
yep, it's very simple. Just turning on a built-in thing